sound/soc/intel/boards/hda_dsp_common.c

```diff
@@ -34,7 +34,7 @@ struct snd_pcm *hda_dsp_hdmi_pcm_handle(struct snd_soc_card *card,
 	return NULL;
 }
 
-#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA_AUDIO_CODEC)
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA_LINK)
```

Comment:
the commit message states an incorrect fact. Look at the sound/soc/intel/boards/Kconfig and you'll see that even I2S machine drivers do rely on HDA_AUDIO_CODEC.

This is the default and it'd probably more that the Chrome configuration is incorrect.